Dockerfile

```diff
@@ -26,13 +17,8 @@ RUN pip install --upgrade pip setuptools && \
     pip install --no-cache-dir -r requirements.txt
 
 COPY . /app
-COPY councilmatic/settings_deployment.py.example /app/councilmatic/settings_deployment.py
 
-# Copy Blackbox executables from builder stage
-COPY --from=builder /usr/local/bin/blackbox* /usr/local/bin/
-COPY --from=builder /usr/local/bin/_blackbox* /usr/local/bin/
-COPY --from=builder /usr/local/bin/_stack_lib.sh /usr/local/bin/
-
-RUN DATABASE_URL='' python manage.py collectstatic --noinput
+RUN mkdir /app/static
```

Comment:
@fgregg You need to bake static files into the image with collectstatic. Heroku dynos are ephemeral, so files collected on release won't persist.

I'm not sure, actually, why we have collect static in our templated release script.

Reply:
isn't this what whitenoise is supposed to address, or is that a terrible lie?

Reply:
No, whitenoise is just a static file server: https://whitenoise.evans.io/en/latest/

There's nothing for it to serve if the files aren't there!

Reply:
got it.